data-hibernate-jpa/src/main/resources/META-INF/native-image/io.micronaut.data/data-hibernate-jpa/reflect-config.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "condition":{"typeReachable":"org.hibernate.id.enhanced.TableGenerator"},
+    "name":"org.hibernate.id.enhanced.LegacyNamingStrategy",
```

Comment:
This should be moved to SQL module or the metadata repository

Reply:
This is just a temporary file until the metadata PR gets approved (https://github.com/oracle/graalvm-reachability-metadata/pull/274) and new version of metadata repo released. Once that happen, I will delete this file.

Reply:
I have just upgraded metadata repo version and deleted `reflect-config.json` file from `data-hibernate-jpa`